src/app/handle-page/change-handle-prefix-page/change-handle-prefix-page.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="container">
```

Comment:
why the empty file? if it's necessary, leave a comment or something to make it clear why it's empty

Reply:
consider adding the same comment to all other instances of empty files

Reply:
Added comments to the *.scss files that they are styling files.